<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWidget</class>
 <widget class="QWidget" name="GameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1727</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TryToSurvive!</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imgs.qrc">
    <normaloff>:/img/ImagesForCapoo/WindowIcon.png</normaloff>:/img/ImagesForCapoo/WindowIcon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">GameWidget
{
background-image: url(:/img/ImagesForCapoo/GameBackGround.jpg);
}
</string>
  </property>
  <widget class="QPushButton" name="Pause">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Continue">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackToRoleSelect">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>180</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>BackToRoleSelect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveAt1AndBack">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>180</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>SaveAt 1 AndBack</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveAt2AndBack">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>180</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>SaveAt 2 AndBack</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveAt3AndBack">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>180</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>SaveAt 3 AndBack</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
